<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentpersonal</class>
 <widget class="QDialog" name="studentpersonal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生界面</string>
  </property>
  <widget class="QTabWidget" name="presonalMsg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="personalmsg">
    <attribute name="title">
     <string>个人信息</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="id">
        <property name="text">
         <string>学号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stuid_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="name">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stuname">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="passwdMan">
    <attribute name="title">
     <string>修改密码</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leNewPwd_chk">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leOldPwd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leNewPwd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>旧密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOk">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>新密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="exam_man">
    <attribute name="title">
     <string>考试安排</string>
    </attribute>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>可选科目</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>441</width>
       <height>101</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>439</width>
        <height>99</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget_examing">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>431</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>已考科目</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>441</width>
       <height>141</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>439</width>
        <height>139</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget_examed">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>431</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
